airflow/providers/google/CHANGELOG.rst

```diff
@@ -26,127 +26,159 @@ Breaking changes
 
 * ``apache-airflow-providers-google uses deprecated Google Ads API V8 (#22111)``
```

Comment:
@potiuk the number is wrong. this is the number of the issue not the PR.
The PR is: https://github.com/apache/airflow/pull/22965